parsers/jscript.c

```diff
@@ -41,6 +42,8 @@
 #define isIdentChar(c) \
 	(isalpha (c) || isdigit (c) || (c) == '$' || \
 		(c) == '@' || (c) == '_' || (c) == '#')
+#define newToken() (objPoolGet(TokenPool))
+#define deleteToken(t) (objPoolPut(TokenPool, (t)))
```

Comment:
style: space before the argument list (in the expansion side, obviously).
Feel free to ignore :)